ROADMAP.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Top-level goals
+
+* Extensibility: custom components, shaders
+* Performance: raycasting (three.js), head tracking on mobile (webvr-polyfill), anything DOM/Custom Elements-related
+* Ecosystem: enable discovery and distribution of assets, components, scenes, snippets, shaders
+
+## `0.2.0`
```

Comment:
CHANGELOG is for what actually ships. ROADMAP is for what is intended to ship. admittedly, the roadmap should perhaps be more of a high-level list.

if we don't want to mention specifics (like bug fixes), then we ought to at least make sure every issue is in the correct milestone (I've moved all the ones I thought were relevant to `0.2.0`).